pyproject.toml

```diff
@@ -106,6 +106,8 @@ filterwarnings = [
     'ignore:Unclosed <MemoryObjectSendStream:ResourceWarning',
     # see https://github.com/python/cpython/issues/126259
     'ignore:unclosed database in <sqlite3.Connection object',
+    # remove when this is merged: https://github.com/jupyter-server/jupyter_server/pull/1481
+    'ignore:The `version` property of an event schema must be a string'
```

Comment:
I think it has been released?

Reply:
OK thanks, so I'll update ~`"jupyter_server>=2.11.1,<3.0.0"`~ to `"jupyter_server>=2.15.0,<3.0.0"` in `jupyter_server_ydoc` and remove this filter.

Reply:
I'll update it in main too.

Reply:
Thanks!